backend/frontend/templatetags/custom_tags.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+
+register = template.Library()
+
+
+@register.inclusion_tag("frontend/user/board/partial_user_avatar.html")
+def user_avatar(name, many=False):
```

Comment:
What is `many`? I dont think you are using this argument anywhere. What is it meant to do?

And why does the user avatar take in a name argument? Why not just a user? Why does the calling code need to know that the avtar is derived from the user's name? WHy is that logic scattered everywhere?

Should we even use the user's name? Who knows? If we needed to change that logic we should change it only once. It shouldn't be in every single template. This must be DRY and cohesive.

Reply:
`Many` was used to conditionally style a group of user avatars. But I moved that to the template.

The user_avatar tag now takes in a user. The user profile is likely to live on the user model, so the only thing that would need updating once we have profile pictures is the user avatar template and the context for the template. We won't need to change all the templates where the user avatar is rendered.